.env.sample

```diff
@@ -63,3 +63,28 @@ USER_MANAGEMENT_BASE_URL = ""
 # API DOC
 APIDOC_URL = ""
 APIDOC_PATH = ""
+
+#cassandra Config
```

Comment:
@rakeshSgr - All in caps please for this and all other variables below

Reply:
resolved